charts/csi-baremetal-operator/templates/rbac.yaml

```diff
@@ -38,6 +38,12 @@ rules:
   - "*"
   verbs:
   - "*"
+- apiGroups:
```

Comment:
Please add comments that these parameters required for OpenShift only. Don't we need to use platform flag here as well?

Reply:
Comment added. We use platform flag in csi-baremetal-deployment and it won't work for operator chart. Or we have to add that flag here too

Reply:
Ideally we need to pass _platform_ parameter to the operator. Operator will use it later when deploying CSI.

Reply:
Also, we use platform parameter in `patcher-configmap`, which creates with Deployment chart. Should we move configmap deploying into operator code or into operator helm chart?

Reply:
Could you please create issue for this and add TODO in the code to address in the next pull request?

Reply:
TODO issue was added